shortnames/shortnames.conf

```diff
@@ -32,6 +32,8 @@
   "mistral:7b-v3" = "huggingface://MaziyarPanahi/Mistral-7B-Instruct-v0.3-GGUF/Mistral-7B-Instruct-v0.3.Q4_K_M.gguf"
   "mistral:7b-v2" = "huggingface://TheBloke/Mistral-7B-Instruct-v0.2-GGUF/mistral-7b-instruct-v0.2.Q4_K_M.gguf"
   "mistral:7b-v1" = "huggingface://TheBloke/Mistral-7B-Instruct-v0.1-GGUF/mistral-7b-instruct-v0.1.Q5_K_M.gguf"
+  "mistral-small3.1" = "hf://bartowski/mistralai_Mistral-Small-3.1-24B-Instruct-2503-GGUF/mistralai_Mistral-Small-3.1-24B-Instruct-2503-IQ2_M.gguf"
```

Comment:
How about just mistral-small?

Reply:
It's deliberately named this, so users don't download:

https://ollama.com/library/mistral-small3.1

we can do both of course

Reply:
mistral-small on Ollama is actually 3.0 rather than 3.1

https://ollama.com/library/mistral-small

Reply:
This bartowski namespace in general is a good goto on huggingface, it's very popular

Reply:
```
mistral-small on Ollama is actually 3.0 rather than 3.1

https://ollama.com/library/mistral-small
```

Isn't this likely to change, and at that point mistral-small would pull a broken Model?

Reply:
Most often this doesn't change, I don't know exactly why, but companies just tend to create new names for models rather than changing the existing ones (I guess part of the reason is the trend to use the tag for x billion parameters, etc.)

Reply:
Some versions of models can be a complete flop also and companies actually recommend people to use the older version